app/Console/Services/Importer.php

```diff
@@ -405,15 +417,40 @@ protected function importPireps()
 
     protected function importUsers()
     {
-        /*$this->comment('--- USER IMPORT ---');
+        $this->comment('--- USER IMPORT ---');
 
         $count = 0;
         foreach ($this->readRows('pilots') as $row)
         {
+            // TODO: What to do about pilot ids
 
+            $name = $row->firstname.' '.$row->lastname;
+            $airlineid = Airline::where('icao', $row->code)->first()->id;
+            $rankid = Rank::where('name', $row->rank)->first()->id;
+            $state = $this->getUserState($row->retired);
+            $attrs = [
+                'name' => $name,
+                'email' => $row->email,
+                'password' => "",
+                'api_key' => "",
+                'airline_id' => $airlineid,
+                'rank_id' => $rankid,
+                'home_airport_id' => $row->hub,
+                'curr_airport_id' => $row->hub,
+                'flights' => intval($row->totalflights),
+                'flight_time' => intval($row->totalhours),
```

Comment:
This actually needs to be converted to minutes. There's a `Utils::hoursToMinutes()`